content/content-tool/tool/src/webapp/vm/content/sakai_filepicker_attach.vm

```diff
@@ -100,10 +100,12 @@ includeLatestJQuery('sakai_filepicker_attach.vm');
 	</script>
 
 	<script>
-		document.addEventListener("DOMContentLoaded", function(event){ 
-			$('#resourceTabs a').click(function (e) {
-				e.preventDefault();
-				$(this).tab('show');
+		document.addEventListener('DOMContentLoaded', () => {
+			document.querySelectorAll('#resourceTabs .nav-link').forEach(link => {
+				link.addEventListener('click', e => {
+					e.preventDefault();
+					new bootstrap.Tab(link).show();
+				});
```

Comment:
I have a question about all of this. Why is bootstrap not doing this for us? You are effectively doing what bootstrap's js should be doing. What am I missing?